### PR TITLE
added fightbot

### PR DESCRIPTION
**Pull request by MrDiamond64**
basically aimbot copy and pasted but with some additional features

### CODE REVIEW

**Comment by Intoprelised** [comment]
nothing happenes for me

**Comment by NRGJobro** [comment]
I just made something like this literally yesterday lmao. It was a .joe fight option lmao. It basicly goes to the nearest person, and turns on killaura lol

**Comment by Intoprelised** [comment]
horion used to have .path player

**Comment by Intoprelised** [comment]
i shall make that

**Comment by NRGJobro** [comment]
I showed the player option again, so u can do .joe player NRG

**Comment by KasinoK** [comment]
Can someone point me in the right direction to get the updated client? or how i can update it to the current version myself?

**Comment by Intoprelised** [comment]
> Can someone point me in the right direction to get the updated client? or how i can update it to the current version myself?

find sigs, offsets, use vtables and update broken code 🤗

**Comment by MrDiamond64** [comment]
> 
> 
> nothing happenes for me

fixed.

**Comment by NRGJobro** [comment]
> Can someone point me in the right direction to get the updated client? or how i can update it to the current version myself?

About all of the sigs have been found. What we need now is the offsets. If you want more info, contact floppy.

**Comment by Intoprelised** [comment]
@MrDiamond64 
Horion is dead, FloppyDolphin is creating new Continued horion repository and it will be made public soon
also whats ur discord?

**Comment by MrDiamond64** [comment]
> @MrDiamond64 
> 
> Horion is dead, FloppyDolphin is creating new Continued horion repository and it will be made public soon
> 
> also whats ur discord?

|| ☭ Uͨ̊̆Ś̋̾S͌̈̓R̊͐ͪ ☭||#9732

gl trying to add it

File: Horion/Module/Modules/FightBot.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#include "FightBot.h"
+
+FightBot::FightBot() : IModule(0, Category::COMBAT, "Moves and attacks at the closest player") {
+	this->registerFloatSetting("Range", &this->range, this->range, 3.f, 8.f);
```
Comment (Intoprelised) [inline Horion/Module/Modules/FightBot.cpp:4]: Id turn up range to 100

**Pull request closed without merging**